web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1255,6 +1258,29 @@ goog.require('gn_alert');
         return !onMdView && gnGlobalSettings.gnCfg.mods.footer.showSocialBarInFooter;
       };
 
+      function getPortals() {
+        var url = '../api/sources';
```

Comment:
Could we use http://localhost:8080/geonetwork/srv/api/sources/subportal directly to only get portals ? Here we need `srv` and portals only.

Reply:
I guess we can, if this `api` call returns the portal and subportals

Reply:
I see it only returns subportals (make sense if you look at the name), but where do you get info about the default portal (the `srv` one)?